test/cypress/integration/consumer-tools/ask-cfpb/answer-helpers.cy.js

```diff
@@ -3,6 +3,14 @@ export class AskCfpbAnswerPage {
     cy.visit('/ask-cfpb/').get('.ask-categories article li a').first().click();
   }
 
+  getSummaryContentLink() {
+    return cy.get('.o-summary_content a').first();
+  }
+
+  getSummaryBtn() {
+    return cy.get('.o-summary_btn');
+  }
+
   clickSummary() {
     cy.get('.o-summary_btn').click();
```

Comment:
Should this be changed to use the new `getSummaryBtn()` to avoid duplication?

Reply:
Oh actually this can be deleted. I went with `getSummaryBtn().click()` in the tests as I didn't think this helper added much over that. Thanks for flagging.